infra/cifuzz/cifuzz.py

```diff
@@ -80,11 +79,14 @@
     level=logging.DEBUG)
 
 
+# pylint: disable=too-many-arguments
```

Comment:
Note to self: `# pylint: disable=too-many-arguments,too-many-locals` and make it local to the function and not file-wide.

Reply:
@evverx I don't think you need to fix issues with the original PR.